setup.py

```diff
@@ -125,6 +125,9 @@ def read(*names, **kwargs):
         'bz2file >= 0.98',
         'text-unidecode >= 1.0, < 2.0',
 
+        # extractcode and scancode
```

Comment:
This is for plugins at large not for ` extractcode and scancode` ... Also you might want to put an upper bound on the pluggy version (e.g. 0.5.0)